amazon-kinesis-client/src/test/java/software/amazon/kinesis/coordinator/SchedulerTest.java

```diff
@@ -292,6 +297,39 @@ public final void testCleanupShardConsumers() {
         assertFalse(shardConsumer1.isShutdownRequested());
     }
 
+    @Test
+    public final void testParentShardDeletedAtShardEndHappyCase() {
+        Set<String> childShards = new HashSet<>();
+        childShards.add("child-0");
+        childShards.add("child-1");
+
+        final Lease lease = new Lease("leaseKey", "leaseOwner", 0L, UUID.randomUUID(),
+               0L, ExtendedSequenceNumber.TRIM_HORIZON, null, 0L,
+                Collections.emptySet(), childShards, null, null)
+                
+        for (Lease childShardLease : childShardLeases) {
```

Comment:
where the childShardLeases is defined ?

Reply:
This got accidentally pushed in the diff when adding unit tests, will have this fixed in next revision.